pybossa/view/fileproxy.py

```diff
@@ -93,11 +107,53 @@ def encrypted_file(store, bucket, project_id, path):
     cipher = AESWithGCM(secret)
     decrypted = cipher.decrypt(content)
 
-    ## respond
     response = Response(decrypted, content_type=_key.content_type)
-
     response.headers.add('Content-Encoding', _key.content_encoding)
     response.headers.add('Content-Disposition', _key.content_disposition)
-    response.headers.add('Cache-Control', 'no-store')
-    response.headers.add('Pragma', 'no-cache')
     return response
+
+
+@blueprint.route('/hdfs/<string:cluster>/<int:project_id>/<path:path>')
```

Comment:
I'm guessing this route wouldn't be used for the public instance.  Do we want to somehow disable routes like this in that situation?

Reply:
we can